Gemfile

```diff
@@ -1,11 +1,14 @@
 source 'https://rubygems.org'
 
 gem 'sinatra'
+gem 'bootstrap', '~> 5.0.1'
+
 
 group :test do
   gem 'capybara'
   gem 'rspec'
   gem 'rubocop', '0.79.0'
   gem 'simplecov', require: false
   gem 'simplecov-console', require: false
+  gem 'launchy'
```

Comment:
What does launchy do?

Reply:
It allows it to open the web browser automatically every time you do rspec, I actually should've got rid of this later on